### PR TITLE
updateing credits and CHANGELOG

### PR DESCRIPTION
**Pull request by tobiasd**
this shall be updated in case there are updates before the release. So keep it open till then.

### CODE REVIEW

**Comment by Hypolite** [comment]
Thanks for the mention!

**Comment by tobiasd** [comment]
sure thing :-)

File: CHANGELOG [hunk 1]
```diff
@@ -10,15 +10,14 @@ Version 3.5.1
     Improvements to the database handling [heluecht]
     Improvements to the admin panel [tobiasd, Hypolite]
     Improvements to the update process [heluecht]
-    Improvements to the federation as contact to Mastodon users can now be established [heluecht, fabrixxm]
```
Comment (annando) [inline CHANGELOG]: The contact can be established - only the communication doesn't work :-)

Comment (tobiasd) [inline CHANGELOG]: I know, I'll add it again once it works ;-)

**Comment by annando** [comment]
Why have you closed it?

**Comment by tobiasd** [comment]
I like to break things. Especially git ;-) and hece the PR got closed when the branch was deleted. There will be a new one in a sec.

**Pull request closed without merging**